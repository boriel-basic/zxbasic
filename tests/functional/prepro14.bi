' Must return just PRINT "HELLO WORLD"


#define y PRINT "HELLO WORLD"
#define test x
#define x y

test


