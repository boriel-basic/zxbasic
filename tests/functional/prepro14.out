#line 1 "prepro14.bi"







PRINT "HELLO WORLD"


